examples/tools/bevymark.rs

```diff
@@ -238,20 +251,25 @@ fn collision_system(windows: Res<Windows>, mut bird_query: Query<(&mut Bird, &Tr
         if y_pos + HALF_BIRD_SIZE > half_height && y_vel > 0.0 {
             bird.velocity.y = 0.0;
         }
-    }
+    });
 }
 
 fn counter_system(
     diagnostics: Res<Diagnostics>,
     counter: Res<BevyCounter>,
-    mut query: Query<&mut Text>,
+    mut query: Query<&mut Text, With<StatsText>>,
 ) {
+    let mut text = query.single_mut();
+
+    if counter.is_changed() {
+        text.sections[1].value.clear();
```

Comment:
I think we should call this optimization in a brief comment like
```
// clearing and writing rather than assigning avoids an allocation
```
(if that's accurate)

Reply:
Done.